tests/integration/states/test_resource_group.py

```diff
@@ -36,7 +37,8 @@ def test_present(salt_call_cli, resource_group, location, connection_auth):
     assert data == expected
 
 
-@pytest.mark.run(order=1, after="test_present", before="test_absent")
+# @pytest.mark.run(order=1, after="test_present", before="test_absent")
+@pytest.mark.skip(reason="resource group already exists")
 def test_changes(salt_call_cli, resource_group, location, tags, connection_auth):
```

Comment:
**issue (blocking):** I think we can still keep some "changes" tests in here for the resource group. maybe a `test_changes_add_tag` and `test_changes_remove_tag` to add and remove the same tag on the resource group so it's net unchanged after the tests run?